producer/bdtpolicy.go

```diff
@@ -58,7 +58,7 @@ func getBDTPolicyContextProcedure(bdtPolicyID string) (
 	} else {
 		// not found
 		problemDetail := util.GetProblemDetail("Can't find bdtPolicyID related resource", util.CONTEXT_NOT_FOUND)
-		logger.Bdtpolicylog.Warnf(problemDetail.Detail)
+		logger.Bdtpolicylog.Warnln(problemDetail.Detail)
```

Comment:
I am curious, why this change?

Reply:
In fails when we run the linter. As there is not non-constant string, we need to use  Warnln. If we use f-string, the we need to Warnf.

producer/bdtpolicy.go:61:29: printf: non-constant format string in call to (*github.com/sirupsen/logrus.Entry).Warnf (govet)
                logger.Bdtpolicylog.Warnf(problemDetail.Detail)